contrib/dist/release_checklist_issue_template

```diff
@@ -0,0 +1,102 @@
+       - Pro tip: if this is a new release series (i.e., this is `vx.y.0` where `y>1`, or this is `vx.0.0`), you will need to be more creative in examining the git logs because this release is on a different branch than the prior release (`vx.(y-1).z`).  Hence, `git log ... last_release_tag..this_branch_name` will not necessarily give you need.  You may need to merge what has changed on your branch with what has changed on the prior release branch, depending on when the prior release branched from this branch.  Read the **SPECIFYING RANGES** section `gitrevisions(7)` for more details.
+  - [ ] `LICENSE`: Update the years in the copyright notices
+  - [ ] ...any other doc files that may not be included in this list
+- [ ] Make final `openmpi-x.y.z.tar.gz` and `openmpi-x.y.z.bz2` tarballs using the _make_dist_tarball_ script in contrib/dist. To do this correctly, you'll need to
+      -  log in to aws.open-mpi.org and su mpiteam
+      -  module load (one of the autconf module files)
+      -  git clone the Open MPI, cd into ompi, switch to the right branch, then
+      -  ./contrib/dist/make_dist_tarball
+
+## Ensure that the tarball actually works
+- [ ] Ensure openmpi `make distcheck` passes
+- Build and install Open MPI (from a tarball)
+  - [ ] Check that the version number and release date is correct in the installed man pages
+- [ ] Build and run the examples in the tarball
+  - [ ] MPI / C bindings
+  - [ ] MPI / C++ bindings
+  - [ ] MPI / All three Fortran bindings (you'll need newer compiler, e.g. gcc 4.9 or higher)
+  - [ ] OSHMEM / C bindings (don't run if not using ikrit)
+  - [ ] OSHMEM / Fortran bindings (don't run if not using ikrit)
+- [ ] Build and install the IBM tests from the `ompi-tests` repo
+  - [ ] Ensure the tests pass with at least 1 or 2 different MPI transports
+- [ ] With a prior release tarball from the same release series (if applicable):
+  - [ ] Build and install Open MPI
+  - [ ] Build all the examples in the tarball (against the prior OMPI release)
+  - [ ] Build the IBM tests from the `ompi-tests` repo (against the prior OMPI release)
+  - [ ] `rm -rf` the installation of the prior release
+  - [ ] Set `LD_LIBRARY_PATH` to point to the installation of the new Open MPI
+  - [ ] Run examples, make sure they still work
+  - [ ] Run IBM tests, make sure they still work
+
+## Test packaging
+- [ ] Ensure that building an Open MPI source RPM works on a RHEL system
+- [ ] Ensure that building the Open MPI binary RPM works on a RHEL system (note had to do hacking on buildrpm.sh script and it doesn't report correct information about where rpms are installed, at least not on my RHEL7.2 system running as root)
+- [ ] Install both RPMs on an RHEL system
+- [ ] Test building and running the IBM tests against the RPM-installed Open MPI
+- [ ] Test building a MPI-based program with the output from `pkg-config` with the RPM-installed `ompi.pc`
+- [ ] Test that uninstall of the rpm works
```

Comment:
If we're going to have an SRPM, we should test it.  Otherwise, we shouldn't bother having the SRPM.

Users have definitely complained before when the SRM didn't work for one reason or another.

If we want to stop distributing an SRPM, that's a conversation we can have.